### PR TITLE
Add Insights API Documentation

### PR DESCRIPTION
**Pull request by sckp**
Adds new pages for Insights and Insights HQ documentation.
Places pages between groups and mailings (alphabetically placed). 

 - Closes [INS-565](https://myemma.atlassian.net/browse/INS-565)

### CODE REVIEW

File: api/external/insights.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+  }
+]
+</pre>
+                            </div>
+                            </p>
+                        </dd></dl>
+                    <dl class="get">
+                        <dt id="get--#account_id-mailings">
+                            <tt class="descname">GET </tt><tt class="descname">/#account_id/insights/v2/audience</tt><a class="headerlink" href="#get--#account_id-mailings" title="Permalink to this definition">&para;</a></dt>
+                        <dd><p>Get information about audience growth by subaccount.</p>
```
Comment (Streeter8) [inline api/external/insights.html]: Should be something more like
`Get information audience growth information for the account`
as this is not the hq endpoint

**Review by Streeter8 — APPROVED** [review]
One small comment, but looks good otherwise!

File: api/external/insights.html (hunk 1 above)
Comment (sckp) [reply api/external/insights.html]: Thank you for the catch!!

File: api/external/insights.html [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  }
+]
+</pre>
+                            </div>
+                            </p>
+                        </dd></dl>
+                    <dl class="get">
+                        <dt id="get--#account_id-mailings">
+                            <tt class="descname">GET </tt><tt class="descname">/#account_id/insights/v2/audience</tt><a class="headerlink" href="#get--#account_id-mailings" title="Permalink to this definition">&para;</a></dt>
+                        <dd><p>Get information audience growth information for the account.</p>
```
Comment (bhenry) [inline api/external/insights.html:186]: typo got transferred here. verbiage feels weird. i think an "about" got lost. or maybe drop the first "information"?

**Pull request closed without merging**